modules/ROOT/pages/release-notes/release-notes54.adoc

```diff
@@ -29,38 +30,39 @@ The {productname} 5.4 release includes the following improvements for the table
 
 * Adds: commands, APIs, and icons for; cut, copy, and paste columns.
 * Adds toolbar button icons for the cut, copy, and paste rows.
-* Adds a new link:{baseurl}/plugins/opensource/table/#commands[`mceTableApplyCellStyle`] command for applying selected styles to table cells.
-* Adds a new link:{baseurl}/plugins/opensource/table/#table_header_type[`table_header_type`] option for setting the HTML structure used for table header rows.
+* Adds a new xref:plugins/opensource/table.adoc#commands[`mceTableApplyCellStyle`] command for applying selected styles to table cells.
```

Comment:
This xref doesn't resolve as the `commands` id is missing on the relevant heading in `table.adoc`. e.g:
```
[[commands]]
== Commands
```

Reply:
done